apps/students/views.py

```diff
@@ -93,37 +93,37 @@ def attendence(request):
         class_range_min, class_range_max = selected_class.split('-')
 
         # Mark everyone's absent
-        stu_att_today = StudentAttendence.objects.filter(
+        stu_att_today = StudentAttendance.objects.filter(
             cal_date=today_cal, student__school_class__range=(class_range_min, class_range_max))
         for stu_att in stu_att_today:
             stu_att.present = False
             stu_att.save()
 
         for stu_id in stu_array:
-            stu_att = StudentAttendence.objects.get(student__id=stu_id, cal_date=today_date)
+            stu_att = StudentAttendance.objects.get(student__id=stu_id, cal_date=today_date)
             stu_att.present = True
             stu_att.save()
 
-        messages.success(request, 'Attendence marked successfully!')
-        return redirect('students:attendence')
+        messages.success(request, 'Attendance marked successfully!')
+        return redirect('students:attendance')
```

Comment:
First line is fine as it is just a text. For the second line, make sure you've changed the name in `urls.py` file also.